dev-requirements.txt

```diff
@@ -1,6 +0,0 @@
-# to run tests ensure you have installed reco-utils with necessary optional dependencies
```

Comment:
it would be nice to have something for external contributors to use to setup an environment to run tests and format the code based on contribution guidelines. if we want to reduce files we can push that into setup.py.

Reply:
sounds good I'll make the changes

Reply:
Pr here: https://github.com/microsoft/recommenders/pull/1445